_episodes/02-image-basics.md

```diff
@@ -97,50 +60,141 @@ color, but that each pixel can have a different color from its neighbors.
 Viewed from a distance, these pixels seem to blend together to form the image
 we see.
 
-## Coordinate system
+## Working with Pixels
+As noted, in practice, real world images will typically be made up of a vast number of pixesl, and each of these pixels will be  one of potentially millions of colors. While we will deal with pictures of such complexity shortly, lets start our exploration with just 15 pixels in a 5 X 3 matrix with 2 colors and work our way up to that complexity, but first the necessary imports.
+
+ ~~~
+"""
+ * Python libraries for learning and performing image processing.
+ *
+"""
+import numpy as np
+import skimage.io
+import skimage.viewer
+import matplotlib.pyplot as plt
+~~~
+{: .language-python}
+
+Now that we have our libraries loaded, let's load our image data from disk.
+ ~~~
+eight = np.loadtxt('data/eight.csv',delimiter=',')
```

Comment:
Instead of using a csv file, perhaps we could generate an image file? So people don't get confused by yet another file format?

Reply:
I am onboard to try using an image file instead, but I am stymied.  Just changing the extension on my existing file doesn't seem to work.  I tried using plt.savefig() to output it, but I get a 640 X 480 X 4 matrix when I reimport that output.  I tried creating it and exporting from photoshop, and I got very close, but I cannot manage to get a matrix that outputs monochrome, I can only get greyscale with 255 for the white squares instead of 1.  Any ideas on how we can get a valid image file with only a 2d matrix with zeros and ones?

Reply:
Okay, seems like skimage.io.imsave can output the array in a format we can use if we use the tif extension.  Tried png, but that led to all kinds of warning and a false color 8 bit image display rather than monochrome.  Given how common tif is in medical applications, I assume we don't want to cut that one completely, so should we just proceed with reading it in as TIF image?